moabb/paradigms/base.py

```diff
@@ -110,8 +110,13 @@ def process_raw(self, raw, dataset, return_epochs=False):
         if len(stim_channels) > 0:
             events = mne.find_events(raw, shortest_event=0, verbose=False)
         else:
-            events, _ = mne.events_from_annotations(raw, event_id=event_id,
-                                                    verbose=False)
+            try:
+                events, _ = mne.events_from_annotations(raw,
+                                                        event_id=event_id,
+                                                        verbose=False)
+            except ValueError:
+                # skip raw if no event found
+                return
```

Comment:
Not sure about this. Why is it interesting to filter out `ValueError`? If there is no event, the error should be raised to stop the pipeline, no?